### PR TITLE
Hardcode IMAX uppercase for Edition Tags

### PR DESCRIPTION
**Pull request by owine**
#### Database Migration
 NO

#### Description
Hardcode IMAX Edition Tags to uppercase - ignore case preference.

#### Todos
- [x] Tests
- [ ] Translation Keys (./src/NzbDrone.Core/Localization/Core/en.json)
- [ ] [Wiki Updates](https://wiki.servarr.com)

#### Issues Fixed or Closed by this PR

### CODE REVIEW

File: src/NzbDrone.Core/Organizer/FileNameBuilder.cs [hunk 1]
```diff
@@ -283,7 +283,11 @@ private string GetLanguageTitle(Movie movie, string isoCodes)
 
         private void AddEditionTagsTokens(Dictionary<string, Func<TokenMatch, string>> tokenHandlers, MovieFile movieFile)
         {
-            if (movieFile.Edition.IsNotNullOrWhiteSpace())
+            if (movieFile.Edition.ToUpper() == "IMAX")
+            {
```
Comment (bakerboy448) [inline src/NzbDrone.Core/Organizer/FileNameBuilder.cs:287]: What about `Extended Theatrical Version Imax`

**Comment by owine** [comment]
Thanks @nuxencs for new logic

File: src/NzbDrone.Core/Organizer/FileNameBuilder.cs [hunk 2]
```diff
@@ -285,7 +285,7 @@ private void AddEditionTagsTokens(Dictionary<string, Func<TokenMatch, string>> t
         {
             if (movieFile.Edition.IsNotNullOrWhiteSpace())
             {
-                tokenHandlers["{Edition Tags}"] = m => CultureInfo.CurrentCulture.TextInfo.ToTitleCase(movieFile.Edition.ToLower());
+                tokenHandlers["{Edition Tags}"] = m => CultureInfo.CurrentCulture.TextInfo.ToTitleCase(movieFile.Edition.ToLower().Replace("imax", "IMAX"));
```
Comment (bakerboy448) [inline src/NzbDrone.Core/Organizer/FileNameBuilder.cs:288]: This is applying IMAX and then converting it to title case

**Review by bakerboy448 — CHANGES_REQUESTED** [review]

**Comment by bakerboy448** [comment]
- [ ] - also needs a test added

**Comment by Qstick** [comment]
We should likely have an edition normalization method in the parser instead of just tacking on a replace in the filenamebuilder. Seems a bit hacky. 

Ideally, we'd normalize more than just IMAX

**Pull request closed without merging**